pkg/apiserver/redirect_test.go

```diff
@@ -71,7 +71,7 @@ func TestRedirect(t *testing.T) {
 		if err == nil || err.(*url.Error).Err != dontFollow {
 			t.Errorf("Unexpected err %#v", err)
 		}
-		if e, a := item.id, resp.Header.Get("Location"); e != a {
+		if e, a := item.id, resp.Header.Get("Location"); "http://"+e != a {
```

Comment:
Move the "http://"+ to before item.id, so that it also prints out in the case of a failure; same below.